packages/react-components/src/components/table/table.tsx

```diff
@@ -4,20 +4,46 @@ import { useTimeSeriesData } from '../../hooks/useTimeSeriesData';
 import { useViewport } from '../../hooks/useViewport';
 import type { StyleSettingsMap, Threshold, Viewport } from '@iot-app-kit/core';
 import { UseCollectionOptions } from '@cloudscape-design/collection-hooks';
-import { TableColumnDefinition, TableItem, TableItemHydrated } from './types';
+import {
+  AlarmItem,
+  TableColumnDefinition,
+  TableItem,
+  TableItemHydrated,
+} from './types';
 import { createTableItems } from './createTableItems';
 import { DEFAULT_TABLE_MESSAGES } from './messages';
 import { TableProps as TableBaseProps } from '@cloudscape-design/components';
 import type { ComponentQuery } from '../../common/chartTypes';
 import { getTimeSeriesQueries } from '../../utils/queries';
+import { useAlarmsFromQueries } from '../../hooks/useAlarmsFromQueries/useAlarmsFromQueries';
+import { parseAlarmStateAssetProperty } from '../../hooks/useAlarms/transformers';
+import { mapAlarmRuleExpression } from '../../hooks/useAlarms/transformers/mapAlarmRuleExpression';
 
 const DEFAULT_VIEWPORT: Viewport = { duration: '10m' };
 
+const ALARM_TABLE_COLUMN_DEFINITIONS: TableColumnDefinition[] = [
+  {
+    key: 'alarmName',
+    header: 'Alarm Name',
+    sortingField: 'alarmName',
+  },
+  {
+    key: 'alarmExpression',
+    header: 'Alarm Expression',
+    sortingField: 'alarmExpressions',
+  },
+  {
+    key: 'state',
+    header: 'State',
+    sortingField: 'state',
+  },
+];
```

Comment:
I'm realizing now we may also want severity to be a column. We can add that in a separate PR though.

Reply:
Added this as a column!